src/app/shared/dynamic-form/components/form-datetime/form-datetime.component.ts

```diff
@@ -0,0 +1,62 @@
+import { AfterViewInit, Component } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { FormBaseComponent } from '../form-base.component';
+import { FormElementBase } from '../../types/form-element-base';
+import { FormActionsService } from '../../../../_services/form-actions.service';
+
+@Component({
+  selector: 'app-form-datetime',
+  templateUrl: './form-datetime.component.html',
+  styleUrls: ['./form-datetime.component.scss'],
+})
+export class FormDatetimeComponent extends FormBaseComponent implements AfterViewInit {
+  config: FormElementBase<string>;
+  group: FormGroup;
+  form: FormGroup;
+  options: any[] = [];
+
+  constructor(public formActionsService: FormActionsService, formBuilder: FormBuilder) {
+    super(formActionsService);
+    this.options = this.timeOptions();
+    this.form = formBuilder.group({
+      date: '',
+      time: '',
+    });
+  }
+
+  ngAfterViewInit(): void {
+    this.form.valueChanges.subscribe(({ date, time }) => {
```

Comment:
please follow the example from dynamic-form-component.ts for the unsubscribing

subscription: Subscription[] = [];

this.subscription.push(
      this.formActionsService.getDiscard().subscribe(() => {

ngOnDestroy(): void {
    this.subscription.forEach((subscription) => subscription.unsubscribe());
  }